.releaserc.cjs

```diff
@@ -157,3 +194,32 @@ function CheckResults(file) {
     numReplacements: 1,
   };
 }
+
+// For dependency replacement.
+// If the runtime defines dependencies,
+// assert the expected number of dependencies were replaced.
+function CheckDependencyReplacementResults(file) {
+  if (file.includes("AwsCryptographicMaterialProviders")) {
+    return {
+      file,
+      hasChanged: true,
+      numMatches: 4,
+      numReplacements: 4,
+    };
+  } else if (file.includes("StandardLibrary")) {
+    return {
+      file,
+      hasChanged: false,
+      numMatches: 0,
+      numReplacements: 0,
+    };
+  }
+  else {
```

Comment:
nit/robustness: handle the known cases explicitly and give an error if the file isn't one of the known cases.